Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -14,14 +14,14 @@
 //  limitations under the License.
 //
 
+#if !os(tvOS)
 #if canImport(LocalAuthentication)
```

Comment:
Let's document what we're doing here, and also coalesce these checks into a single check (I'm pretty sure we can do this but forgive me if I'm wrong):
```suggestion
// Xcode 13 and prior incorrectly say that LocalAuthentication is available on tvOS, so we have to check both as long as Xcode 13 and prior are supported.
#if !os(tvOS) && canImport(LocalAuthentication)
```